study.md

```diff
@@ -45,6 +49,10 @@ help manage view-state?
 
 ```md
 <!-- your answer here -->
+I think most developers would interpret view as what the usually can actually see
+on the browser. I'm not completely sure on this but my interpretation on the view-state refers to the different parts of the page that have
+or have not fully rendered. The routers(such as scrolling) is what is used
+to navigate between the different view-state
 ```
```

Comment:
View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state.